torch/_dynamo/exc.py

```diff
@@ -227,32 +239,21 @@ def filter_stack(stack):
 
 
 def format_error_msg_verbose(exc, code, record_filename=None, frame=None):
-    msg = str(
-        format_bytecode(
-            "WON'T CONVERT",
-            code.co_name,
-            code.co_filename,
-            code.co_firstlineno,
-            code,
-        )
```

Comment:
I have never found the bytecode printout here useful. Got rid of it. cc @williamwen42